### PR TITLE
README.md: Add lab.toml example

### PR DESCRIPTION
**Pull request by prarit**
Add an example lab.toml file.

Fix: 805

Signed-off-by: Prarit Bhargava <prarit@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ instance. There are several ways to provide this information to `lab`:
 3. local configuration file in [Tom's Obvious, Minimal Language (TOML)](https://github.com/toml-lang/toml): `./lab.toml`;
     - No other config files will be used as overrides if a local configuration file is specified
 4. user-specific configuration file in TOML: `~/.config/lab/lab.toml`.
+    - an example of user-specfic configurations in lab.toml can be found here.  As can be seen in the example file, any command option can be set in the configuration file.
```
Comment (bmeneg) [inline README.md:94]: Isn't a link to `docs/lab.toml` missing in the word `here`?

**Review by bmeneg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (prarit) [reply README.md:94]: Oops.  Sorry, this was meant to be a draft!  I was going to play with the output a bit ... but yes, it will be a link.

Comment (bmeneg) [reply README.md]: Oh, my bad. I overlooked the "draft" state!
Sorry for that :))

**Comment by prarit** [comment]
@bmeneg what do you think of it 'inline' https://github.com/prarit/lab/tree/lab_toml_example ?

**Comment by bmeneg** [comment]
@prarit I think it makes sense.

**Comment by prarit** [comment]
> @prarit I think it makes sense.

Okay, marking as ready.

File: docs/lab.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+[mr_checkout]
+  force = true
+
+[mr_create]
+  force-linebreak = true
+  draft = true
+
+[mr_edit]
+  force-linebreak = true
+
```
Comment (bmeneg) [inline docs/lab.toml:15]: Since we're keeping the example inline shouldn't this file go away?

**Review by bmeneg — CHANGES_REQUESTED** [review]

**Review by bmeneg — APPROVED** [review]
Yep, this is it :))

**Pull request merged**